### PR TITLE
Additional API fixes 

### PR DESCRIPTION
**Pull request by Dudblockman**
Cleaning up parts I meant to do in the last API update

### CODE REVIEW

**Comment by Dudblockman** [comment]
I think there might be a better way to handle the returnvalue of ItemCAD.cast, perhaps Optional?

File: src/main/java/vazkii/psi/common/item/ItemCAD.java [hunk 1]
```diff
@@ -78,12 +78,7 @@
 import javax.annotation.Nonnull;
 import javax.annotation.Nullable;
 
-import java.util.Arrays;
-import java.util.Collections;
-import java.util.HashSet;
-import java.util.List;
-import java.util.Optional;
-import java.util.Set;
+import java.util.*;
```
Comment (Kamefrede) [inline src/main/java/vazkii/psi/common/item/ItemCAD.java:81]: please don't use wildcard imports

File: src/main/java/vazkii/psi/common/item/ItemCAD.java [hunk 2]
```diff
@@ -202,7 +197,7 @@ public ActionResultType onItemUse(ItemUseContext ctx) {
 				setCADComponent(playerCad, dyeStack);
 			}
 		}
-		boolean did = cast(worldIn, playerIn, data, bullet, itemStackIn, 40, 25, 0.5F, ctx -> ctx.castFrom = hand);
+		boolean did = null != cast(worldIn, playerIn, data, bullet, itemStackIn, 40, 25, 0.5F, ctx -> ctx.castFrom = hand);
```
Comment (Kamefrede) [inline src/main/java/vazkii/psi/common/item/ItemCAD.java]: nitpick but change the order of the condition

**Review by Kamefrede — CHANGES_REQUESTED** [review]

File: src/main/java/vazkii/psi/common/item/ItemCAD.java (hunk 2 above)
Comment (Dudblockman) [reply src/main/java/vazkii/psi/common/item/ItemCAD.java]: Already resolved, comment made in reference of outdated commit.

**Pull request merged**